package.json

```diff
@@ -72,7 +72,7 @@
     "postcss-modules": "^4.3.0",
     "postcss-preset-env": "^7.2.0",
     "prettier": "2.8.0",
-    "react-docgen": "^5.4.0",
+    "react-docgen": "^6.0.4",
```

Comment:
v5 was generating syntax error related to `this` keyword, also, it was throwing error when there're multiple component definitions. Upgrading solved those problems.

But the files where those errors were happening are now excluded. So, if we consider that, there's no urge to do that. But I think newer version is usually better, because of the bugfixes and stuff.

Reply:
It's good to keep dependencies upgraded thanks!